docs/install/install-script.md

```diff
@@ -39,10 +39,11 @@ A secret named `model-serving-etcd` will be created and passed to the controller
 
 ## Installation
 
-Download and extract the latest modelmesh-serving from [GitHub](https://github.com/kserve/modelmesh-serving):
+Install the latest release of [modelmesh-serving](https://github.com/kserve/modelmesh-serving) by first cloning the corresponding release branch:
 
 ```shell
-git clone git@github.com:kserve/modelmesh-serving.git
+RELEASE=release-0.8
+git clone -b $RELEASE --depth 1 --single-branch git@github.com:kserve/modelmesh-serving.git
```

Comment:
Shall we use https instead of ssh which requires a public key?
I am getting "git@github.com: Permission denied (publickey)." with the current command.
Using https should work for everyone without additional setup `git clone -b $RELEASE --depth 1 --single-branch https://github.com/kserve/modelmesh-serving.git`

Reply:
Yea, good point. I will change it.